ultraLatency.py

```diff
@@ -94,12 +94,11 @@
 print(startTime)
 Date = startTime
 
-if options.amc13local:
-    import amc13
-    connection_file = "%s/connections.xml"%(os.getenv("GEM_ADDRESS_TABLE_PATH"))
-    amc13base  = "gem.shelf%02d.amc13"%(options.shelf)
-    amc13board = amc13.AMC13(connection_file,"%s.T1"%(amc13base),"%s.T2"%(amc13base))
-    pass
+import amc13
```

Comment:
Does this mean the "--amc13local" option will be depreciated?  If so suggest to remove from list of avaliable cmd options to limit option bloat.